x-pack/plugins/lens/public/shared_components/axis/extent/helpers.ts

```diff
@@ -86,9 +103,39 @@ export function getDataBounds(
   }
 }
 
-export function validateExtent(shouldIncludeZero: boolean, extent?: UnifiedAxisExtentConfig) {
-  return {
-    inclusiveZeroError: shouldIncludeZero && !validateZeroInclusivityExtent(extent),
-    boundaryError: !validateAxisDomain(extent),
-  };
+export function validateExtent(
+  hasBarOrArea: boolean,
+  extent: UnifiedAxisExtentConfig,
+  scaleType?: YScaleType | XScaleType
+): {
+  helpMsg?: string;
+  errorMsg?: string;
+} {
+  if (!validateAxisDomain(extent)) {
+    return {
+      errorMsg: i18n.translate('xpack.lens.axisExtent.boundaryError', {
+        defaultMessage: 'Lower bound has to be less than upper bound',
```

Comment:
In general I think we can simplify these a bit more by using "should".

Lower bound should be less than upper bound.

Reply:
Done 97c62b179f91e8f93d37cd0cd2b962a6bda89231